package.json

```diff
@@ -111,6 +112,7 @@
     "semver": "^5.1.1",
     "silent-error": "^1.0.0",
     "sort-package-json": "^1.4.0",
+    "strip-ansi": "^4.0.0",
```

Comment:
@kellyselden you're right, and that is why @locks apparently changed it to be a dev dep 😉